UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/ReactionManager.cs

```diff
@@ -50,7 +50,7 @@ private void Update()
 						foreach ( var pushable in matrix.Get<PushPull>( windyNode.Position, true ) )
 						{
 							float correctedForce = windyNode.WindForce / ( int ) pushable.Pushable.Size;
-							if ( correctedForce >= AtmosConstants.MinPushForce )
+							if (!pushable.isNotPushable && correctedForce >= AtmosConstants.MinPushForce )
```

Comment:
do you propose to add this check everywhere where push can happen? bad practice
already told you that this check is integrated in tile-snapped Push, but only missing in Nudge that's used for sub-tile pushing

so all you have to do is add this check to CNT.Nudge():
```cs
    public void Nudge( NudgeInfo info ) {
        if ( PushPull.isNotPushable )
        {
            return;
        }
```